apps/web/frontend/components/inviteLink.tsx

```diff
@@ -27,7 +27,9 @@ export const InviteLink = (props: InviteLinkProps) => {
   const [{ fetching: fetchingRegenerateInviteKey }, regenerateInviteKey] = useMutation(
     projectRegenerateInviteKeyMutation,
   )
-  const inviteLink = `${process.env.NEXTAUTH_URL}/projects/join/${project.inviteKey}`
+
+  const nextAuthUrl = process.env.NEXT_PUBLIC_APP_URL
+  const inviteLink = `${nextAuthUrl}/projects/join/${project.inviteKey}`
```

Comment:
Now, it no longer related to next auth

```suggestion
  const inviteLink = `${process.env.NEXT_PUBLIC_APP_URL}/projects/join/${project.inviteKey}`
```